projects/Mox/example.nix

```
@@ -0,0 +1,6 @@
+{ ... }:
+{
+  services.mox.enable = true;
+  services.mox.hostname = "mail";
+  services.mox.user = "admin@example.ke";
```

Maybe we should use another hostname, since (from running `mox quickstart`):

> Quickstart assumes mox will run on the machine you run quickstart on and uses
its host name and public IPs. On many systems the hostname is not a fully
qualified domain name, but only the first dns "label", e.g. "mail" in case of
"mail.example.org". If so, quickstart does a reverse DNS lookup to find the
hostname, and as fallback uses the label plus the domain of the email address
you specified. Use flag -hostname to explicitly specify the hostname mox will
run on.

```suggestion
  services.mox.hostname = "localhost";
  services.mox.user = "admin@localhost";
```